packages/ruleset/src/functions/api-symmetry.js

```diff
@@ -319,9 +364,13 @@ function checkForGraphFragmentPattern(
     false
   );
 
+  logger.debug(
+    `${ruleId}: isGraphFragment() returned [${variantIsGraphFragment},${variantOmitsProperties}]`
+  );
+
   if (!variantOmitsProperties) {
     logger.info(
-      `${ruleId}: schema does not omit any properties in the canonical schema - it is not a proper graph fragment`
+      `${ruleId}: variant schema properties are not a proper subset of the canonical schema properties`
```

Comment:
This is a matter of taste but I still think it helps make it clear to say something about properties not being omitted rather than just indicating it isn't a subset. e.g. it also wouldn't be a proper subset if anything was added, but this check is specifically about not omitting properties and the user should know that's what happened.

I'd be fine with something like this, even:

```suggestion
      `${ruleId}: variant schema properties are not a proper subset of the canonical schema properties (no properties omitted)`
```